tensorboard/webapp/analytics/analytics_test.ts

```diff
@@ -0,0 +1,97 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {TestBed} from '@angular/core/testing';
+import {Store, Action, MemoizedSelector} from '@ngrx/store';
+import {MockStore, provideMockStore} from '@ngrx/store/testing';
+import {provideMockActions} from '@ngrx/effects/testing';
+import {ReplaySubject} from 'rxjs';
+
+import {changePlugin} from '../core/actions';
+import {getActivePlugin} from '../core/store';
+import {State} from '../core/store';
+import {AnalyticsLogger} from './analytics_logger';
+import {AnalyticsEffects} from './effects';
+import {createState, createCoreState} from '../core/testing';
+
+describe('changing plugins', () => {
```

Comment:
`changing plugins` -> `analytics`. You want to know where failing test specs are coming from the reporter.